### PR TITLE
Update pricing to match MAU

### PR DESCRIPTION
**Pull request by michaelmairegger**

### CODE REVIEW

File: docs/auth/index.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Using App Center Auth has the following limitations in preview:
...
+App Center Auth is a free service, but you pay for [Azure AD B2C authentications](https://azure.microsoft.com/pricing/details/active-directory-b2c/). The first 50,000 active monthly users are free. An authentication is defined as a token issued either in response to a sign-in request initiated by a user, or initiated by an application on behalf of a user (e.g. token refresh, where the refresh interval is configurable). It is free to store authenticated users in the Azure AD B2C tenant. If you turn on **Multi-Factor Authentication**, you will be charged at a flat fee of $0.03 per authentication.
```
Comment (mmacy) [inline docs/auth/index.md]:
```suggestion
App Center Auth is a free service, but you pay for [Azure AD B2C authentications](https://azure.microsoft.com/pricing/details/active-directory-b2c/). The first 50,000 monthly active users (MAU) are free. A MAU is counted when a unique user authenticates within a given calendar month. It is free to store authenticated users in the Azure AD B2C tenant. If you turn on **Multi-Factor Authentication**, you will be charged at a flat fee of $0.03 per authentication.
```

**Review by mmacy — COMMENTED** [review]
Thanks for the update, @michaelmairegger! Suggesting a change here that swaps the authentications explanation with MAU explanation.

File: docs/auth/index.md (hunk 1 above)
Comment (mmacy) [reply docs/auth/index.md]: Cc: @amchew

**Comment by michaelmairegger** [comment]
> Thanks for the update, @michaelmairegger! Suggesting a change here that swaps the authentications explanation with MAU explanation.

@mmacy Thanks

**Comment by PRMerger8** [comment]
@michaelmairegger : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by King-of-Spades** [comment]
This doc was deleted, so I'm closing this request.

**Pull request closed without merging**